Runner/MethodExecutor.cs

```diff
@@ -83,12 +92,22 @@ public ProtoExecutionResult ExecuteHooks(string hookType, HooksStrategy strategy
                 builder.Failed = true;
                 var isScreenShotEnabled = Utils.TryReadEnvValue("SCREENSHOT_ON_FAILURE");
                 if (isScreenShotEnabled == null || isScreenShotEnabled.ToLower() != "false")
-                    builder.ScreenShot.Add(TakeScreenshot());
+                {
+                    builder.ScreenShot = TakeScreenshot();
```

Comment:
why do we need both `ScreenShot` and `FailureScreenshot` to be taken? Even if you need both, you can hold the value in a variable instead of invoking `TakeScreenshot` twice